content/v2.0-preview/get-started/get-started-with-managed-resources.md

```diff
@@ -17,66 +20,67 @@ This quickstart requires:
 * AWS [access keys](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-creds)
 
 ## Install the AWS provider
-
-Install the AWS S3 provider into the Kubernetes cluster with a Kubernetes 
-configuration file. 
+Install the AWS S3 provider into the Kubernetes cluster with a Kubernetes
+configuration file.
 
 ```yaml {label="provider",copy-lines="all"}
-cat <<EOF | kubectl apply -f -
 apiVersion: pkg.crossplane.io/v1
 kind: Provider
 metadata:
   name: provider-aws-s3
 spec:
-  package: xpkg.crossplane.io/crossplane-contrib/provider-aws-s3:v1.21.1
-EOF
+  package: xpkg.crossplane.io/crossplane-contrib/provider-aws-s3:v1.22.0-crossplane-v2-preview.0
+```
+
+Save this to a file called `provider.yaml`, then apply it with:
+```shell {label="kube-apply-provider",copy-lines="all"}
+kubectl apply -f provider.yaml
 ```
 
-The Crossplane {{< hover label="provider" line="3" >}}Provider{{</hover>}}
+The Crossplane {{< hover label="provider" line="2" >}}Provider{{</hover>}}
 installs the Kubernetes _Custom Resource Definitions_ (CRDs) representing AWS S3
-services. These CRDs allow you to create AWS resources directly inside 
+services. These CRDs allow you to create AWS resources directly inside
 Kubernetes.
```

Comment:
I'd like to drop all mentions of CRDs here - too much detail for this context IMO.